website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -488,6 +496,7 @@ file. The variables must be set with proper values for your environment in the
 | `db_user_password`     | Password for RDS database user authentication           | "My_Sup3rS3cr3tuserPassw0rd"                                |
 | `dlq_subscription_email`| The email to notify users when messages are received in dead letter SQS queue | "test@email.com"                     |
 | `orca_default_bucket`  | Default ORCA S3 Glacier bucket to use.                  | "PREFIX-orca-primary"                                       |
+| `orca_restore_retrieval_type`  | The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited' | "Bulk"                     |
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
updated